src/Sulu/Bundle/ContentBundle/Document/RouteDocument.php

```diff
@@ -52,6 +52,16 @@ class RouteDocument implements
     private $targetDocument;
 
     /**
+     * @var \DateTime
+     */
+    private $created;
+
+    public function __construct()
+    {
+        $this->created = new \DateTime();
```

Comment:
Because the `TimestampableBehavior` is localized. Maybe we should introduce a `LocalizedTimestampableBehavior` and make the current one not localized.